tests/CMakeLists.txt

```diff
@@ -187,6 +187,9 @@ ConfigureTest(BINNING_MR_TEST mr/device/binning_mr_tests.cpp)
 # callback memory resource tests
 ConfigureTest(CALLBACK_MR_TEST mr/device/callback_mr_tests.cpp)
 
+# system memory resource tests
+ConfigureTest(SYSTEM_MR_TEST mr/device/system_mr_tests.cu)
```

Comment:
I think the `system_memory_resource` should also be tested using the `mr_ref_tests` for comprehensive testing, since it's a core memory resource.

Reply:
Done.